music21/metadata/__init__.py

```diff
@@ -546,26 +546,15 @@ def contributors(self) -> t.List[Contributor]:
         Returns [] if no Contributors exist.
 
         >>> md = metadata.Metadata()
-        >>> md['composer'] = ['Ludwig', 'Wolfgang']
-        >>> md['librettist'] = 'Joe'
-        >>> md.add('architect', 'John')      # 'architect' is new in 2022
-        >>> md.add('title', 'Caveat Emptor') # not a contributor
+        >>> md['composer'] = ['Richard Strauss']
+        >>> md.librettist = 'Oscar Wilde'
+        >>> md.add('title', 'Salome') # not a contributor
```

Comment:
tiny style -- two spaces before an inline comment.  But better to not use inline comments in doctests where you can instead use prose:

```
When we add a title (whether through `.attribute` setting, `[item]` setting, or the :meth:`~music21.metadata.Metadata.add` method), it will not show up in the list of contributors.
```

This is clearer about what is happening.

Reply:
Done.